ui/app/models/gcp/config.js

```diff
@@ -65,8 +69,45 @@ export default class GcpConfig extends Model {
     'identityTokenTtl',
   ];
 
+  get isWifPluginConfigured() {
+    return !!this.identityTokenAudience || !!this.identityTokenTtl || !!this.serviceAccountEmail;
+  }
+
+  get isAccountPluginConfigured() {
+    // credentials is not checked here because it's never return by the API
+    // additionally credentials can be set via GOOGLE_APPLICATION_CREDENTIALS env var so we cannot call it a required field in the ui.
+    // thus we can never say for sure if the account accessType has been configured so we always return false
+    return false;
+  }
```

Comment:
If this is always false then this doesn't need to be a getter and can just be a property on the model. Also it might be helpful to clarify that we're referring to the "credentials" parameter here, since credentials can refer to a set of params in some instances
```suggestion
isAccountPluginConfigured = false
    // credentials is not checked here because it's never return by the API
    // additionally credentials can be set via GOOGLE_APPLICATION_CREDENTIALS env var so we cannot call it a required field in the ui.
    // thus we can never say for sure if the account accessType has been configured so we always return false
```

Reply:
I like it. Thanks.